sdks/go/pkg/beam/core/runtime/harness/monitoring_test.go

```diff
@@ -0,0 +1,122 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package harness
+
+import (
+	"testing"
+
+	"github.com/apache/beam/sdks/go/pkg/beam/core/metrics"
+)
+
+func TestGetShortID(t *testing.T) {
+	tests := []struct {
+		id           string
+		urn          mUrn
+		typ          mType
+		expectedUrn  string
+		expectedType string
+	}{
+		{
+			id:           "1",
+			urn:          urnUser,
```

Comment:
Can you add the case where the same urn but unique labels are used gets a different short id?